spacy_llm/backends/rest/backend/openai.py

```diff
@@ -97,42 +97,54 @@ def __call__(self, prompts: Iterable[str]) -> Iterable[str]:
         }
         api_responses: List[str] = []
         prompts = list(prompts)
-
-        data: Dict[str, Union[List[str], List[Dict[str, str]]]] = {}
         url = self._url if self._url else self.supported_models[self._config["model"]]
```

Comment:
This is neither here nor there, but you could do:

```python
url = self._url or self.supported_models[self._config["model"]]
```

Reply:
Your suggestion is more elegant, I suspect it might also be a bit less clear? I personally prefer it, but I've had negative feedback in the past about being too clever with conditionals.

Not changing this now, but I'll keep it in mind!